diff --git a/modules/editor/file-templates/templates/c++-mode/__cpp b/modules/editor/file-templates/templates/c++-mode/__cpp
deleted file mode 100644
index 10f8293e1..000000000
--- a/modules/editor/file-templates/templates/c++-mode/__cpp
+++ /dev/null
@@ -1,7 +0,0 @@
-# -*- mode: snippet -*-
-# group: file templates
-# contributor: Henrik Lissner
-# --
-#include "`(file-name-nondirectory (file-name-sans-extension (buffer-file-name)))`.h"
-
-$0
diff --git a/modules/editor/file-templates/templates/c++-mode/__hpp b/modules/editor/file-templates/templates/c++-mode/__hpp
deleted file mode 100644
index c38ae152c..000000000
--- a/modules/editor/file-templates/templates/c++-mode/__hpp
+++ /dev/null
@@ -1,10 +0,0 @@
-# -*- mode: snippet -*-
-# group: file templates
-# contributor: Henrik Lissner
-# --
-#ifndef ${1:__`(upcase (file-name-base buffer-file-name))`_H_$(upcase yas-text)}
-#define $1
-
-$0
-
-#endif // $1
diff --git a/modules/editor/file-templates/templates/c++-mode/__main.cpp b/modules/editor/file-templates/templates/c++-mode/__main.cpp
deleted file mode 100644
index b4a1557a8..000000000
--- a/modules/editor/file-templates/templates/c++-mode/__main.cpp
+++ /dev/null
@@ -1,13 +0,0 @@
-# -*- mode: snippet -*-
-# group: file templates
-# contributor: Henrik Lissner
-# --
-#include <iostream>
-
-using namespace std;
-
-int main(int argc, char *argv[]) {
-    $0
-
-    return 0;
-}
diff --git a/modules/editor/file-templates/templates/c++-mode/__winmain.cpp b/modules/editor/file-templates/templates/c++-mode/__winmain.cpp
deleted file mode 100644
index 67947be3b..000000000
--- a/modules/editor/file-templates/templates/c++-mode/__winmain.cpp
+++ /dev/null
@@ -1,12 +0,0 @@
-# -*- mode: snippet -*-
-# group: file templates
-# contributor: Henrik Lissner
-# --
-#include <Windows.h>
-
-int CALLBACK WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine, int nCmdShow)
-{
-    $0
-
-    return 0;
-}
diff --git a/modules/lang/org/config.el b/modules/lang/org/config.el
index acd05aa29..b46923c9f 100644
--- a/modules/lang/org/config.el
+++ b/modules/lang/org/config.el
@@ -141,18 +141,18 @@ This forces it to read the background before rendering."
         '((sequence
            "TODO(t)"  ; A task that needs doing & is ready to do
            "PROJ(p)"  ; A project, which usually contains other tasks
-           "STRT(s)"  ; A task that is in progress
-           "WAIT(w)"  ; Something external is holding up this task
-           "HOLD(h)"  ; This task is paused/on hold because of me
+           "STRT(s@)"  ; A task that is in progress
+           "WAIT(w!)"  ; Something external is holding up this task
+           "HOLD(h@)"  ; This task is paused/on hold because of me
            "|"
-           "DONE(d)"  ; Task successfully completed
-           "KILL(k)") ; Task was cancelled, aborted or is no longer applicable
+           "DONE(d@)"  ; Task successfully completed
+           "KILL(k@)") ; Task was cancelled, aborted or is no longer applicable
           (sequence
-           "[ ](T)"   ; A task that needs doing
-           "[-](S)"   ; Task is in progress
-           "[?](W)"   ; Task is being held up or paused
+           "[ ](T!)"   ; A task that needs doing
+           "[-](S!)"   ; Task is in progress
+           "[?](W!)"   ; Task is being held up or paused
            "|"
-           "[X](D)")) ; Task was completed
+           "[X](D!)")) ; Task was completed
         org-todo-keyword-faces
         '(("[-]"  . +org-todo-active)
           ("STRT" . +org-todo-active)
